charts/agent-control/values.yaml

```diff
@@ -33,7 +33,7 @@ agent-control-deployment:
   image:
     registry:
     repository: newrelic/newrelic-agent-control
-    tag: "nightly"
+    tag: "0.26.3"
```

Comment:
Where is this coming from? I remember seen this change or similar in another PR.

Reply:
discussed offline